SwiftUI_DZ/SwiftUI_DZ/View/StartedView.swift

```diff
@@ -0,0 +1,174 @@
+            .fontWeight(.bold)
+            .multilineTextAlignment(.center)
+            .padding()
+    }
+
+    private var navigationToDescriptionView: some View {
+        NavigationLink(
+            destination: DescriptionView(),
+            tag: Constants.descriptionViewTagString,
+            selection: $tagSelection
+        ) { EmptyView() }
+    }
+
+    private var furnitureImageView: some View {
+        AsyncImage(url: URL(string: furnitures.randomElement()?.imagePath ?? Constants.emptyString)) { phase in
+
+            switch phase {
+            case .empty:
+                ProgressView()
+                    .accentColor(.accentColor)
+            case let .success(image):
+                image
+                    .resizable()
+                    .scaledToFill()
+                    .frame(
+                        width: Constants.phaseImageWidth,
+                        height: Constants.phaseImageHeight
+                    )
+            case let .failure(error):
+                VStack {
+                    Image(systemName: Constants.wifiSystemImageName)
+                    Text("\(error.localizedDescription)")
+                        .font(.headline)
+                }
+            @unknown default:
+                fatalError()
+            }
+        }
+    }
+
+    private var getStartedButtonView: some View {
+        Button {
+            tagSelection = Constants.descriptionViewTagString
+        } label: {
+            Text(Constants.getStartedButtonText.uppercased())
+                .foregroundColor(.purple)
+                .font(.title2)
+                .fontWeight(.bold)
+        }
+        .padding()
+        .background(
+            RoundedRectangle(cornerRadius: Constants.buttonRectangleCornerRadius)
+
+                .fill(LinearGradient(
+                    colors: [.purple.opacity(Constants.purpleOpacity), .red],
+                    startPoint: .leading,
+                    endPoint: .trailing
+                ))
+                .frame(width: Constants.getStartedButtonRectangleFrame)
+        )
+    }
+
+    private var haveAccountTextView: some View {
+        Text(Constants.haveAccountText)
+            .foregroundColor(.white)
+            .font(.headline)
+            .fontWeight(.bold)
+    }
+
+    private var singHereButtonView: some View {
+        Button {
+            tagSelection = Constants.singUpViewTagString
+        } label: {
+            Text(Constants.singButtonText)
+                .foregroundColor(.white)
+                .font(.title)
+                .bold()
+        }
+        .background(
+            RoundedRectangle(cornerRadius: 0)
+                .foregroundColor(.white)
+                .frame(
+                    width: Constants.singHereRectangleWidth,
+                    height: Constants.singHereRectangleHeight
+                )
+                .padding(.top, Constants.singHereButtonPadding)
+        )
+        .padding()
+    }
+
+    /// Превью
```

Comment:
```suggestion
    /// Превью
```
 Женя многим удалял, спросим потом у него, может тут не обязательно  описание, для превью экрана